src/pages/developing/svelte-tutorial/step-2.mdx

```diff
@@ -0,0 +1,719 @@
+`DataTable`.
+
+At this point, we need to render a static data table in `repos.svelte`.
+
+### Render data table
+
+We'll be setting the `rows` array from an API in the next tutorial step, but for
+now, static example rows will suffice.
+
+Add the following data to the `headers` and `rows` variables:
+
+```html path=src/routes/repos.svelte
+<script>
+  import { Row, Column, DataTable } from 'carbon-components-svelte';
+
+  const headers = [
+    { key: 'name', value: 'Name' },
+    { key: 'createdAt', value: 'Created' },
+    { key: 'updatedAt', value: 'Updated' },
+    { key: 'issueCount', value: 'Open Issues' },
+    { key: 'stars', value: 'Stars' },
+    { key: 'links', value: 'Links' },
+  ];
+
+  const rows = [
+    {
+      id: '1',
+      name: 'Repo 1',
+      createdAt: 'Date',
+      updatedAt: 'Date',
+      issueCount: '123',
+      stars: '456',
+      links: 'Links',
+    },
+    {
+      id: '2',
+      name: 'Repo 2',
+      createdAt: 'Date',
+      updatedAt: 'Date',
+      issueCount: '123',
+      stars: '456',
+      links: 'Links',
+    },
+    {
+      id: '3',
+      name: 'Repo 3',
+      createdAt: 'Date',
+      updatedAt: 'Date',
+      issueCount: '123',
+      stars: '456',
+      links: 'Links',
+    },
+  ];
+</script>
+
+<Row class="repo-page__r1">
+  <Column>
+    <DataTable
+      expandable
+      title="Carbon Repositories"
+      description="A collection of public Carbon repositories."
+      {headers}
+      {rows}
+    >
+      <p slot="expanded-row" let:row>Row description</p>
+    </DataTable>
+  </Column>
+</Row>
+```
+
+## Style repo page
+
+Our styles for the repo page are mostly fine. We just need to update a few
+vertical spacing issues.
+
+Add a `style` block to the file, add the following styles:
+
+```html path=src/routes/repos.svelte
+<style>
+  :global(.repo-page__r1) {
+    padding-top: 'spacing-05';
+    padding-bottom: 'spacing-05';
+  }
+</style>
+```
+
+Congratulations! We've now created our static repo page!
+
+## Submit pull request
+
+We're going to submit a pull request to verify completion of this tutorial step.
+
+### Continuous integration (CI) check
+
+Run the CI check to make sure we're all set to submit a pull request.
+
+```bash
+yarn ci-check
+```
+
+<InlineNotification>
+
+**Note:** Having issues running the CI check?
+[Step 1](</developing/svelte-tutorial/step-1#continuous-integration-(ci)-check>)
+has troubleshooting notes that may help.
+
+</InlineNotification>
+
+### Git commit and push
+
+Before we can create a pull request, stage and commit all of your changes:
+
+```bash
+git add --all && git commit -m "feat(tutorial): complete step 2"
+```
+
+Then, push to your repository:
+
+```bash
+git push origin svelte-step-2
+```
+
+<InlineNotification>
+
+**Note:** Having issues pushing your changes?
+[Step 1](/developing/svelte-tutorial/step-1#git-commit-and-push) has
+troubleshooting notes that may help.
+
+</InlineNotification>
+
+### Pull request (PR)
```

Comment:
Opened PR but stuck in "👷 Deploy request for carbon-tutorial-svelte pending review." https://github.com/carbon-design-system/carbon-tutorial-svelte/pull/14